initrd/options.go

```diff
@@ -4,10 +4,15 @@
 // You may not use this file except in compliance with the License.
 package initrd
 
+import (
+	"path/filepath"
+)
+
 type InitrdOptions struct {
-	output   string
-	cacheDir string
-	arch     string
+	output          string
+	cacheDir        string
+	arch            string
+	kraftignorePath string
```

Comment:
I think this option, `kraftignorePath`, can be removed and we can _always_ enable `.kraftignore` for relevant initrd providers. If the file exists the relevant procedure is accounted for.

Reply:
Done 👍🏽

Reply:
Done 👍🏽